tests/unit/authenticators/jwt-test.js

```diff
@@ -496,15 +514,15 @@ test('#refreshAccessToken triggers the `sessionDataUpdated` event on successful
 
 test('#getTokenData returns correct data', function() {
   var jwt = JWT.create();
-  
+
```

Comment:
sorry about the white space but my editor automatically removes trailing white space